scripts/misc.js

```diff
@@ -135,59 +135,43 @@ export function createQR(strData = '', domImg, size = 4) {
     domImg.firstChild.style.borderRadius = '8px';
 }
 
-//generate private key for masternodes
-export async function generateMnPrivkey() {
-    // maximum value for a decoded private key
-    let max_decoded_value =
+/**
+ * Generate an encoded private key for masternodes
+ */
+export function generateMnPrivkey() {
```

Comment:
This whole function can just be replaced with getSafeRand(32), the probability of it being out of the range is so small it's not worth the lines of code

Reply:
You agree @panleone? I'll make this change if so.

Reply:
the probability is indeed very small!
The probability of a failing is around 10^38/10^77